src/hotspot/cpu/riscv/stubRoutines_riscv.cpp

```diff
@@ -55,4 +55,13 @@ address StubRoutines::riscv::_string_indexof_linear_ul = nullptr;
 address StubRoutines::riscv::_large_byte_array_inflate = nullptr;
 address StubRoutines::riscv::_method_entry_barrier = nullptr;
 
+ATTRIBUTE_ALIGNED(64) const jint StubRoutines::riscv::_arrays_hashcode_powers_of_31[] =
+{
+         923521, // 0x000E1781
+          29791, // 0x0000745F
+            961, // 0x000003C1
```

Comment:
based on the comment above about `pow31_2 `, line 62-64 can be removed now.

Reply:
Done.